### ISSUE TITLE
Error: Cannot find module 'combined-stream

### ISSUE THREAD

**Issue by juztinlazaro** [googlemaps/google-maps-services-js#994]
Error: Cannot find module 'combined-stream'\nRequire stack:\n- /var/task/node_modules/@googlemaps/google-maps-services-js/node_modules/form-data/lib/form_data.js

![image](https://github.com/googlemaps/google-maps-services-js/assets/10995629/fa5d321d-28d0-4941-8e4b-503d81ae8ffb)

Please be sure to include as much information as possible:

#### Environment details

1. zipcode: `3333`
2. win10
3. nodejs, serverless

#### Code example

```nodejs
# example

import GoogleMaps from '@googlemaps/google-maps-services-js';

let googleMapsClient;

function createGoogleMapsClient() {
  if (googleMapsClient) {
    return googleMapsClient;
  }
  googleMapsClient = GoogleMaps.createClient({
    key: process.env.GOOGLE_PLACES_API_KEY,
  });

  return googleMapsClient;
}

export default createGoogleMapsClient;


export function placesAutoComplete(query, options = {}) {
  console.log('placesAutoComplete===');
  const components = {};
  if (options.countryCode)
    components.country =
      options.countryCode; /* can't pass undefined as country */
  return createGoogleMapsClient()
    .places.Autocomplete({
      input: query,
      components,
      types: options.types,
    })
    .then((response) => response.json.predictions);
}
```

Following these steps will guarantee the quickest resolution possible.

Thanks!

**Comment by wangela** [comment]
If you would like to upvote the priority of this issue, please comment below or react with :+1: so we can see what is popular when we triage.
--------

@juztinlazaro Thank you for opening this issue. 🙏
Please check out these other resources that might help you get to a resolution in the meantime:

- [Check the issue tracker](https://developers.google.com/maps/support/#issue_tracker) - bugs and feature requests for Google Maps Platform APIs and SDKs
- [Open a support case](https://developers.google.com/maps/support#contact-maps-support) - Get 1:1 support in Cloud Console.
- [Discord](https://discord.gg/fHTDmZe9En) - chat with other developers
- [StackOverflow](http://stackoverflow.com/questions/tagged/google-maps) - use the `google-maps` tag


_This is an automated message, feel free to ignore._

**Comment by juztinlazaro** [comment]
@amuramoto any thoughts?

**Comment by amuramoto** [comment]
When are you seeing this crash? Is it on start up? Sometime during runtime when the code above is called?

**Comment by juztinlazaro** [comment]
@amuramoto yes in startup, FYI im using nodeJS and we deployed it in AWS lambda, already wiping node_modules and reinstalling the packages.

**Comment by usefulthink** [comment]
Is this issue still occuring in the latest version? 

The `combined-stream` package is a transitive dependency of `axios` (via `form-data`), and should definitely be installed together with axios. If it's not, there's something bad going on that we can't really help debug.
